### DISCUSSION TITLE
Is there a way to change VM data directory path in settings?

### DISCUSSION THREAD

**Discussion by dirk68-work** [macstadium/orka-desktop#6] [Q&A]
Hi,
I'm trying to change to path where the VM data resides using the settings menu. Unfortunately the dropdown "VM data directory" is disabled and always points. to ~/.local/share/orka/data.
How do I change it?

**Comment by abojak** [comment]
Greetings Dirk!
Orka Desktop does not yet support changing the source directory for VMs
within the application. We are about to launch a new product "Orka Engine"
(Orka anywhere) that offers CLI control of VMs and comes as a local
installer package as well. That product will merge the source directory
with Orka Desktop. After that release, we can consider supporting user
control of the source VM directory. Orka Engine releases in November of
this year.

Regards.
Andrew Bojak
MacStadium
Orka PM

**Reply by 1stqualifier** [reply]
Once you get it integrated I'll definitely give orka desktop a try.

**Reply by mgruenthal** [reply]
Ditto! Would be very helpful to move VMs to an external drive.

**Reply by mgruenthal** [reply]
In the meantime, I've worked around this limitation by moving the share folder containing VMs to an external drive and creating a symbolic link (e.g. ln -s /path to new location /Users/me/.local)
